site/en/docs/extensions/mv3/service_workers/index.md

```diff
@@ -3,14 +3,15 @@ layout: 'layouts/doc-post.njk'
 title: About extension service workers
 description: Extension service workers are an extension's central event handler. That makes them different from web service workers. 
 date: 2023-05-03
+update: 2023-07-26
 ---
 
 This section explains what you need to know to use service workers in extensions. You should read this section whether you're familiar with service workers or not. Extension service workers are an extension's central event handler. That makes them just different enough from web service workers that the mountains of service worker articles around the web may or may not be useful. 
 
 Extension service workers have a few things in common with their web counterparts. An extension service worker is loaded when it is needed, and unloaded when it goes dormant. Once loaded, an extension service worker generally runs as long as it is
 actively receiving events, though it [can shut down](/docs/extensions/mv3/service_workers/service-worker-lifecycle/#idle-shutdown). Like its web counterpart, an extension service worker cannot access the DOM, though you can use it if needed with [offscreen documents](/docs/extensions/reference/offscreen/).
 
-Extension service workers are more than network proxies (as web service workers are often described). In addition to the [standard service worker events](https://developer.mozilla.org/docs/Web/API/ServiceWorkerGlobalScope#events), they also respond to extension events such as navigating to a new page, clicking a notification, or closing a tab. They're also registered and updated differently from web service workers.
+Extension service workers have some important differences with web service workers. First, they're are more than network proxies (as web service workers are often described). In addition to the [standard service worker events](https://developer.mozilla.org/docs/Web/API/ServiceWorkerGlobalScope#events), they also respond to extension events such as navigating to a new page, clicking a notification, or closing a tab. In some cases, these APIs can extend the service worker's lifetime or even start a service worker in response to specific events. `chrome.tabs.onCreated` does this, for example. A few APIs can keep a service worker alive indefinitely. An example of this is communicating with a [native host](/docs/extensions/mv3/nativeMessaging/). Finally, extension service workers are registered and updated differently from web service workers.
```

Comment:
Hmm, I don't think "In some cases, these APIs can [...] even start a service worker in response to specific events." makes sense. Don't all extension events wake up the service worker?

Reply:
I should have said "some APIs can extend..."

Reply:
Would you be ok just removing that line? I'm actually not sure it's helpful here given we haven't introduced the idea of a lifetime yet. I feel like trying to include it means we then have to try to get really precise about the language in what is supposed to be an intro 😅

Reply:
+1 to Oliver